source/gloperate/source/stages/base/FramebufferStage.cpp

```diff
@@ -67,30 +64,41 @@ void FramebufferStage::rebuildFBO()
         }
 
         for (auto input : this->inputs()) {
-            if (auto slot = dynamic_cast<gloperate::InputSlot<globjects::Texture *> *>(input)) {
-                // Get texture (if set)
-                globjects::Texture * texture = slot->data();
-                if (texture && i == 0) {
-                    if (slot->name() != "texDepth") {
-                        // Add color attachment
-                        colorAttachments.push_back(index);
-                        index = index + 1;
-                    }
-                } else if (texture && i == 1) {
-                    if (slot->name() == "texDepth") {
-                        // Add depth attachment
-                        m_fbo->attachTexture(gl::GL_DEPTH_ATTACHMENT, texture);
-                    } else {
-                        // Add color attachment
-                        m_fbo->attachTexture(index, texture);
-                        index = index + 1;
-                    }
+            auto slot = dynamic_cast<Input<RenderTarget *> *>(input);
+            if (!slot) {
+                continue;
+            }
+            RenderTarget * texture = **slot;
```

Comment:
Now you're officially a 3-star programmer :wink: 
Source: http://wiki.c2.com/?ThreeStarProgrammer